index.html

```diff
@@ -0,0 +1,187 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Joaquin's Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap" crossorigin="anonymous" rel="stylesheet">
+</head>
+<body>
+    <header>
+        <nav class="flexnav">
+            <h3>Welcome</h3>
+            <a href="" class="menu">≡</a>
+        </nav>
+    </header>
+    <section class="main">
+        <section class="background">
+        <svg width="375" height="650" viewBox="0 0 375 650" fill="none" xmlns="http://www.w3.org/2000/svg">
+            <path d="M-71.4806 468.094L44.3004 393.777" stroke="#A5ADBA" stroke-width="0.426199" stroke-miterlimit="10" stroke-linecap="round" stroke-dasharray="0.43 0.43"/>
+            <path d="M-105.984 368.193L25.6795 342.365" stroke="#A5ADBA" stroke-width="0.426199" stroke-miterlimit="10" stroke-linecap="round" stroke-dasharray="0.43 0.43"/>
+            <path d="M-107.628 261.104L24.0361 286.932" stroke="#A5ADBA" stroke-width="0.426199" stroke-miterlimit="10" stroke-linecap="round" stroke-dasharray="0.43 0.43"/>
+            <path d="M-76.3007 159.862L39.4802 234.179" stroke="#A5ADBA" stroke-width="0.426199" stroke-miterlimit="10" stroke-linecap="round" stroke-dasharray="0.43 0.43"/>
+            <path d="M-15.7264 76.6514L70.2604 190.564" stroke="#A5ADBA" stroke-width="0.426199" stroke-miterlimit="10" stroke-linecap="round" stroke-dasharray="0.43 0.43"/>
+            <path d="M66.8653 21.583L112.542 161.202" stroke="#A5ADBA" stroke-width="0.426199" stroke-miterlimit="10" stroke-linecap="round" stroke-dasharray="0.43 0.43"/>
```

Comment:
Kindly add the image as a background image in your CSS file instead of HTML file. You can use the `background` class in CSS to do this.